CHANGELOG.md

```diff
@@ -36,6 +36,7 @@ Changed:
 - The 'app.cash.redwood.generator.compose.protocol' and 'app.cash.redwood.generator.widget.protocol' Gradle plugins are now deprecated and will be removed in the next release. Use 'app.cash.redwood.generator.protocol.guest' and 'app.cash.redwood.generator.protocol.host', respectively.
 - The 'redwood-tooling-codegen' CLI flags for protocol codegen have changed from `--compose-protocol` and `--widget-protocol` to `--protocol-guest` and `--protocol-host`, respectively.
 - Entrypoints to the protocol on the host-side and guest-side now require supplying the version of Redwood in use on the other side in order to ensure compatibility and work around any bugs in older versions. This uses a new `RedwoodVersion` type, and will be automatically wired if using our Treehouse artifacts.
+- In `UIViewLazyList`'s `UITableView`, adding special-case handling for programmatic scroll-to-top calls
```

Comment:
This needs moved up into the "Unreleased" section. Also, period at the end, please!

Reply:
Fixed! Think this might have gotten caught in a rebase.